deploying-monitoring-redis-oke.md

```diff
@@ -0,0 +1,257 @@
+redis-metrics-794db76ff7-xmd2q   1/1     Running   0          156m                                                                               
+redis-slave-7fd8b55f7-25w8d      1/1     Running   1          156m                                                                               
+redis-slave-7fd8b55f7-hvhmc      1/1     Running   1          156m                                                                               
+redis-slave-7fd8b55f7-mjq8q      1/1     Running   1          156m
+```
+
+Afterwards, use port-forward to so you can access it using the redis-cli:
+
+<!--EDIT is there actually a command `k`?-->
+```console
+k -n redis port-forward redis-master-0 6379:6379
+Forwarding from 127.0.0.1:6379 -> 6379
+```
+
+Open a new terminal, login into the bastion and obtain the Redis password:
+
+```console
+export REDIS_PASSWORD=$(kubectl get secret --namespace redis redis -o jsonpath="{.data.redis-password}" | base64 --decode)
+```
+
+Do a quick test to see you can connect to Redis:
+
+<!--EDIT command appears mangled-->
+```console
+redis-cli -a $REDIS_PASSWORD127.0.0.1:6379> ping                                                                                                                                                        
+PONG                                                                                                                                                                        
+127.0.0.1:6379>
+```
+
+Before we import the csv, access Grafana (http://localhost:3000) as described above by opening a third terminal and running kubectl port-forward locally. Browse to the Redis Dashboard and set the refresh to every 5 seconds:
+
+```console
+kubectl -n monitoring port-forward prom-operator-grafana-77cdf86d94-m8pv5 3000:3000
+```
+
+Now import the csv file as follows:
+
+<!--EDIT appears mangled, assume line break in --pipe\nAll, and no line break before file.csv-->
+```console
+awk -F, 'NR > 1{ print "SET", "\"employee_"$1"\"", "\""$0"\"" }' file.csv | redis-cli -a $REDIS_PASSWORD --pipe
+All data transferred. Waiting for the last reply...
+
+Last reply received from server.
+
+errors: 0, replies: 1000000
+```
+
+You may want to watch the Redis dashboard in Grafana. You can see the immediate jump in Network IO, the number of items in the DB as well as the amount of memory used.
+
+{% imgx assets/redis-dashboard-after-insertion.png  "Redis Dashboard after mass insertion" %}
+
+While we installed the Prometheus Operator and Redis Cluster manually using the cli, you can also achieve that using the Terraform helm provider. As you are enabling monitoring on Redis, you need to ensure the relevant CRDs are created. When you are doing that manually and in the order above, this is done for you.
+
+However, when you use Terraform to do the provisioning, you will need to explicitly set the order as follows:
+
+<!--EDIT please verify indentation-->
```

Comment:
Seems like everything after the first line should be outdented one tab-width, but I defer to article's original formatting. Please verify.

Reply:
For this one, I actually did match it to article's formatting (though I did add a space I missed btwn the first code block and the next line).